lib/KBox/src/tasks/NMEA2000Task.cpp

```diff
@@ -113,6 +113,8 @@ void NMEA2000Task::visit(const IMUMessage &m) {
     sendN2kMessage(n2kmessage);
 
     _imuSequence++;
+  } else {
+       DEBUG("IMU is not calibrated: move the unit around to calibrate it"); //debug message added to warn user if IMU not calibrated
```

Comment:
We could show a message like "Calibrating" in place of "APMode: Off" on the nav page until IMU is calibrated.  Could also prevent trying to use the AP until calibration is OK